phone/src/Phone.tsx

```diff
@@ -104,9 +104,10 @@ function Phone() {
               id="phone"
               className="PhoneScreen"
               style={{
-                backgroundImage: settings.customWallpaper
-                  ? `url(${settings.customWallpaper})`
-                  : `url(media/backgrounds/${settings.wallpaper.value})`,
+                backgroundImage:
+                  settings.wallpaper.label === t('APPS_SETTINGS_OPTIONS_CUSTOM_WALLPAPER')
```

Comment:
This creates an edge case where if the user changes their language after setting a wallpaper, the translation will not *match* the `settings.wallpaper.label` in localStorage. This is why I had previously suggested the utility function for determining whether a wallpaper value was a default or not.

A small hack for this would be to detect whether the `settings.wallpaper.value` is a url, as the default wallpapers will not store a URL. Not ideal but it would work if you don't want to use the utility function I suggested.

Edit: This is assuming you meant to make the label a translation key in the setter as well in `WallpaperModal.tsx`